teams/account-management/README.md

```diff
@@ -1,5 +1,54 @@
 # Account Management
 
+## Setup your local dev environment
+
+To get started on local development and testing:
+
+1. **Get your `buffer-dev` environment setup**
+  → https://github.com/bufferapp/buffer-dev
+
+2. **Install the latest version of `yarn`**
+  → [Installing Yarn](https://yarnpkg.com/en/docs/install)
+
+3. **Make sure you have node with version <= 9 (Node v10 is not compatible)**
+    ```
+    $ node -v
+    ```
+
+4. **Install Packages and Bootstrap**
+    ```bash
+    $ cd ~/buffer-dev/buffer-login  # Or wherever yours is located
+    $ yarn
+    $ yarn run bootstrap
+
+    $ cd ~/buffer-dev/buffer-account  # Or wherever yours is located
+    $ yarn
+    $ yarn run bootstrap
+   ```
+
+5. **Start up the docker containers**
+    ```bash
+    $ cd ~/buffer-dev # Or wherever yours is located
+    $ ./dev up session-service-dev core-authentication-service-dev login-dev account publish marketing web
```

Comment:
@hharnisc - is account needed here? I think so. I was getting a ENOTFOUND error for a bit but it seems to be working again 🤔

Reply:
Hey @philippemiguet if you want to develop just the `account` app. I would recommend this command.

```bash
./dev up session-service core-authentication-service login account
```

This way you run only services you need. You run them all if you like, but chances are you're only working on one application at a time :wink: